lib/myxql/protocol.ex

```diff
@@ -513,6 +520,11 @@ defmodule MyXQL.Protocol do
     {:cont, :initial, {:many, [resultset | results]}}
   end
 
+  defp decode_resultset(<<0xFB, rest::binary>>, _next_data, :initial, _row_decoder) do
+    {filename, ""} = take_string_nul(rest)
```

Comment:
Base on the changes to `take_string_nul`, it seems you are not expecting a `nul` here? So maybe we should not call/change said function?

Reply:
You're right, we shouldn't change the function!